packages/analytics-browser/src/config.ts

```diff
@@ -140,12 +141,12 @@ export const useBrowserConfig = async (
   const cookieName = getCookieName(apiKey);
   const cookies = await cookieStorage.get(cookieName);
   const queryParams = getQueryParams();
-  const sessionTimeout = options?.sessionTimeout ?? defaultConfig.sessionTimeout;
+  const sessionManager = await new SessionManager(cookieStorage, apiKey).load();
```

Comment:
session manager instantiation moved to `useBrowserConfig` so we can do async ops. also calling load() through chaining